tests/dummy/app/pods/components/frost-create-form/component.js

```diff
@@ -0,0 +1,69 @@
+import Ember from 'ember'
+import layout from './template'
+
+export default Ember.Component.extend({
+  layout,
+  modalForms: Ember.inject.service('modal-forms'),
+  bunsenModel: {
```

Comment:
Let's not call it bunsen when we use it - this is a `userAccountModel` or something like that